controllers/provider-openstack/pkg/controller/controlplane/valuesprovider.go

```diff
@@ -240,6 +246,16 @@ func getConfigChartValues(
 		"requestTimeout":    cluster.CloudProfile.Spec.OpenStack.RequestTimeout,
 	}
 
+	// Only the 1.15 out of tree provider for OpenStack support the OS_DOMAIN_NAME/ID
+	// https://github.com/kubernetes/cloud-provider-openstack/pull/733
+	ok, err := gutils.CompareVersions(cluster.Shoot.Spec.Kubernetes.Version, ">=", "1.15")
+	if ok {
+		values = map[string]interface{}{
```

Comment:
Why are you overwriting the complete `values` map?

Reply:
That was a mistake. Wanted to add those keys.